package/com.unity.formats.usd/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changes in usd-unitysdk for Unity
 
+## [3.0.0-exp.2] - 2021-09-29
+### Features
+- All interpolation types are now properly supported for Mesh standard attributes (normals, tangents, uvs, colors).
+
+### Bug Fixes
+- Fixed the import of facevarying UVs which showed seams on Meshes.
```

Comment:
Fixed the import of facevarying UVs containing seams ?
 The current sentence does not look very english.